user/pages/03.Howtos/10.l3-high-availability/docs.en.md

```diff
@@ -289,6 +298,280 @@ Aug 07 09:06:18 ha-second Keepalived_vrrp[8665]: VRRP_Instance(vrrp_1) Transitio
+
+```shell
+openstack port create --network ha_lab --fixed-ip subnet=9a027a94-e688-45cb-a653-e223ccc9f72f,ip-address=10.200.51.10 --mac-address 00:00:5e:00:01:01 --security-group <sec_group_1> --security-group <sec_group_2> vip
+```
+
+`ha_lab` network ports list with new 'vip' port:
+
+```shell
+$ openstack port list --network ha_lab
++--------------------------------------+------+-------------------+-----------------------------------------------------------------------------+--------+
+| ID                                   | Name | MAC Address       | Fixed IP Addresses                                                          | Status |
++--------------------------------------+------+-------------------+-----------------------------------------------------------------------------+--------+
+| db8fa14d-5800-4db5-8ad7-fe9ad1eaef37 |      | fa:16:3e:08:e6:15 | ip_address='10.200.51.34', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
+| 913c4131-0cc3-4df4-9f7e-b4cf4ae885ca |      | fa:16:3e:8d:dc:bb | ip_address='10.200.51.32', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
+| a8389d5e-6bd0-4368-a6fb-249127f9cdca |      | fa:16:3e:c9:e0:b1 | ip_address='10.200.51.1', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f'  | ACTIVE |
+| d320ed31-afae-334f-fe10-522ac148bb1a | vip  | 00:00:5e:00:01:01 | ip_address='10.200.51.10', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
+| e220e2d9-f4de-4941-92e0-640ee352aa4f |      | fa:16:3e:f3:a9:a3 | ip_address='10.200.51.33', subnet_id='9a027a94-e688-45cb-a653-e223ccc9f72f' | ACTIVE |
++--------------------------------------+------+-------------------+-----------------------------------------------------------------------------+--------+
+```
+
+Let's install and configure keepalived on `ha_first` and `ha_second` VMs:
+
+```shell
+ubuntu@ha-first:~$ sudo apt install keepalived
+```
+
+Now, let's create the keepalived configuration.
+
+`/etc/keepalived/keepalived.conf` on `ha_first`:
+
+```shell
+vrrp_instance vrrp_1 {
+    debug 2
+    interface ens3
+    state MASTER
+    virtual_router_id 1
+    garp_master_refresh 5
+    garp_master_refresh_repeat 2
+    priority 200
+    unicast_src_ip 10.200.51.32 dev ens3
+    unicast_peer {
+        10.200.51.33 dev ens3
+    }
+    virtual_ipaddress {
+        10.200.51.10
+    }
+}
+```
+
+`/etc/keepalived/keepalived.conf` on `ha_second`:
+
+```shell
+vrrp_instance vrrp_1 {
+    debug 2
+    interface ens3
+    state MASTER
+    virtual_router_id 1
+    garp_master_refresh 5
+    garp_master_refresh_repeat 2
+    priority 100
+    unicast_src_ip 10.200.51.33 dev ens3
+    unicast_peer {
+        10.200.51.32 dev ens3
+    }
+    virtual_ipaddress {
+        10.200.51.10
+    }
+}
+```
+
+Some of these options need special attention in cloud environments:
+
+* `garp_master_refresh 5` - forces MASTER VRRP node to generate GARP packets every 5 seconds to update ARP tables in SDN and on all VMs connected to the network.
```

Comment:
i see you added these options here, what about the other options from previous version ( like use_vmac and vmac_xmit_base)? They worked with cbk/dbl shouldn`t they also be mentioned somewhere?

Reply:
vmac is not working on OVN so it shouldn't be used on FES

Reply:
We could deliberately mark the options as `cbk and dbl only` / `fes only` in a comment to make it explicit that they haven't been forgotten by accident.